CONTRIBUTING.md

```diff
@@ -128,13 +128,14 @@ We highly recommend installing `pre-commit`, which will take care of running Bla
 
 ```bash
 pip install pre-commit
+pre-commit install
 ```
 
 And that's all you need to do from now on.
 
 ### Documentation style
 
-JAX-GalSim follows the NumPy/SciPy format: https://numpydoc.readthedocs.io/en/latest/format.html
+JAX-GalSim follows the NumPy/SciPy format: <https://numpydoc.readthedocs.io/en/latest/format.html>
```

Comment:
Could you explain what is the purpose of doing this?

Reply:
This is just to make the link clickable (I have a markdown formatter that adds this by default)

Reply:
OK, thanks